### ISSUE TITLE
Evaluation on ScanNet++

### ISSUE THREAD

**Issue by mtaktash** [lck666666/plana3r#5]
Hi! 

I was wondering if you used the whole train/val set of ScanNet++ to train the model, or not. I wanted to evaluate the model on the validation set of ScanNet++

Thanks

**Comment by lck666666** [comment]
Hi, our model using almost all images in ScanNet++ as training dataset, so I recommond you test our model on other benchmarks, thanks!
